tests/bench/bench.pact

```diff
@@ -89,13 +89,18 @@
 
  (defun wrap10 (a) (id (id (id (id (id (id (id (id (id (id a)))))))))))
 
+ (defun wrap10_integer:integer (a:integer) (id_integer (id_integer (id_integer (id_integer (id_integer (id_integer (id_integer (id_integer (id_integer (id_integer a)))))))))))
+
  (defun rep10 (a) (id a) (id a) (id a) (id a) (id a) (id a) (id a) (id a) (id a) (id a))
 
  (defun withr () (with-read bench-accounts "Acct1" { "balance":= b } b))
 
+ (defun accum:integer (xs) (fold (+) 0 xs))
+
```

Comment:
While we're at it, how about some other functions for looking at runtime TC and apps, namely:

- more than one arg
- object tc
- list tc

Reply:
I've added benchmarks for these, and wrote up the results in the PR description. Any other ideas for things we should look at?